setup.py

```diff
@@ -0,0 +1,37 @@
+#/usr/bin/env python
+
+import codecs
+import os
+import sys
+
+from os.path import join, dirname
+from setuptools import setup
+import codecs
+
+
+read = lambda filepath: codecs.open(filepath, 'r', 'utf-8').read()
+
+if 'publish' in sys.argv:
+    os.system('python setup.py sdist upload')
+    sys.exit()
+
+setup(
+    name='flask-session-cookie-manager',
+    version="1.2",
+    description="simple Python script to deal with Flask session cookie",
+    long_description=read(join(dirname(__file__), 'README.md')),
+    keywords='Flask session cookie',
+    author='Wilson Sumanang, Alexandre ZANNI',
+    maintainer='Alexandre ZANNI, TAbdiukov',
+    license='Unknown License',
+    url='https://github.com/TAbdiukov/flask-session-cookie-manager', #change this line tp your repo if PR accepted
```

Comment:
I'll accept it so you can change the URL.

Reply:
Changed, thanks!